docs/documents.md

```diff
@@ -2,58 +2,71 @@
 
 ## Documentation overviews
 
-[GKV Spitzenverband, Pflege (§ 105 SGB XI)]([https://www.gkv-datenaustausch.de/leistungserbringer/pflege/pflege.jsp)
+[GKV Spitzenverband, Pflege (§ 105 SGB XI)](https://www.gkv-datenaustausch.de/leistungserbringer/pflege/pflege.jsp)
 
 [GKV Spitzenverband, Sonstige Leistungserbringer (§ 302 SGB V)](https://www.gkv-datenaustausch.de/leistungserbringer/sonstige_leistungserbringer/sonstige_leistungserbringer.jsp)
 
 ## List of latest evaluated document versions
 
-Pflege, Broschüre:  
+### Pflege
+
+- Pflege, Broschüre:  
 [Information zum elektronischen Abrechnungsverfahren mit den gesetzlichen Pflegekassen](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/20210406_Broschuere_TP6_oA.pdf)  
 Status from **2021-04-06**
 
-Pflege, Technische Anlage 1:  
+#### Regelung der Datenübermittlung nach § 105 Abs. 2 SGB XI Technische Anlage 1
+
+- Pflege, Technische Anlage 1:  
 [Technische Anlage für Abrechnung auf maschinell verwertbaren Datenträgern zur Regelung der Datenübermittlung nach § 105 Abs. 2 SGB XI](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/technische_anlagen_aktuell_2/TA1_3.3_20201126_oA.pdf)  
 Version **3.3** from 2020-11-26, to be applied from **10/2021** 
 
-Pflege, Technische Anlage 1, Anhang 1:  
+- Pflege, Technische Anlage 1, Anhang 1:  
 [Struktur Auftragsdatei](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/technische_anlagen_aktuell_2/20070125_TA1_ANH1.pdf)  
 Version **2.0**, to be applied from **07/2007** 
 
-Pflege, Technische Anlage 1, Anhang 2:  
+- Pflege, Technische Anlage 1, Anhang 2:  
 [Testverfahren](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/technische_anlagen_aktuell_2/20030131_TA1_ANH2.pdf)  
 Version **1.0**, to be applied from **10/2003** 
 
-Pflege, Technische Anlage 1, Anhang 3:  
+- Pflege, Technische Anlage 1, Anhang 3:
```

Comment:
I created line breaks with double spaces at the end of the line to make it easier to skim the list, for example for version numbers. This is now partially deleted. Can we keep the line breaks?

Reply:
I don't understand how double spaces at the end make it easier to skim the list. Is this some special markdown syntax?

Reply:
Yes, a double space at the end of a line is rendered as a line break inside a paragraph or list item. Normally, line breaks in a paragraph are ignored. So it is the line break that makes it easier to skim the list and the double spaces are needed to render a line break inside a paragraph / list item.

Reply:
ok, changed. Didn't know that